onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -959,8 +960,8 @@ SubGraphCollection_t TensorrtExecutionProvider::GetSupportedList(SubGraphCollect
         SubGraphCollection_t next_nodes_list;
         const std::vector<NodeIndex>& subgraph_node_index = graph_viewer->GetNodesInTopologicalOrder();
         next_nodes_list = GetSupportedList(parser_nodes_list, iterations, max_iterations, *graph_viewer, early_termination);
-        for (int i = 0, end = next_nodes_list.size(); i < end; ++i) {
-          for (int j = 0, end = next_nodes_list[i].first.size(); j < end; ++j) {
+        for (int i = 0, end = static_cast<int>(next_nodes_list.size()); i < end; ++i) {
```

Comment:
better to change i from int to size_t instead of casting the value returned by size() ?
size_t is unsigned and int is signed so casting to int can be problematic

Reply:
Changed to use size_t.